app-misc/figlet/figlet-2.2.5-r1.ebuild

```diff
@@ -13,6 +13,12 @@ LICENSE="BSD"
 SLOT="0"
 KEYWORDS="~alpha amd64 arm arm64 ~hppa ~ia64 ~mips ppc ppc64 ~sparc x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos"
 
+src_prepare() {
+	use elibc_musl && sed -e '/_DECLS$/d' -i utf8.h
```

Comment:
I would suggest to apply upstream patch https://github.com/cmatsuoka/figlet/commit/88428a728e1f99dcdc63d7c5ff0cd8b6cda37576 with `PATCHES=( ... )` (see [documentation](https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html) for more information) rather than this.